apps/src/codebridge/Workspace/index.tsx

```diff
@@ -48,31 +50,61 @@ const Workspace = () => {
       rightHeaderContent={<HeaderButtons />}
       className={moduleStyles.workspace}
     >
-      <FileTabs />
-      {isStartMode && (
+      <div
+        className={moduleStyles.workspaceWorkarea}
+        style={{
```

Comment:
Could we flip between class names depending on the `showFileBrowser` value rather than hard-code styles here? (Same question for other hard-coded styles)

Reply:
Done!

I tweaked it so it'll tack on a `withFileBrowser` class if `showFileBrowser` is `true`.